README.md

```diff
@@ -1,125 +1,118 @@
 
-![](https://gistcdn.githack.com/jorgebucaran/00f6d3f301483a01a00e836eb17a2b3e/raw/26625256b5e5ccb632f990727db70055ae24e584/nvm.fish.svg)
+- [XDG Base Directory](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) friendly.
+- `.node-version` and `.nvmrc` support. ✅
+- <kbd>Tab</kbd>-completable seamless shell integration.
+- No dependencies, no setup, no clutter—it just works.
+  <!-- - Hot symlink switching—absolute speed unlocked. -->
+    <!-- - Automatic version switching on `$PWD` change. -->
 
 ## Installation
 
-Install with [Fisher](https://github.com/jorgebucaran/fisher) (recommended):
+Install with [Fisher](https://github.com/jorgebucaran/fisher):
 
 ```console
 fisher install jorgebucaran/nvm.fish
 ```
 
-<details>
-<summary>Not using a package manager?</summary>
+## Quickstart
 
-###
+Install the latest Node release and start using it.
 
-Copy [`conf.d/nvm.fish`](conf.d/nvm.fish), [`functions/nvm.fish`](functions/nvm.fish), and [`completions/nvm.fish`](completions/nvm.fish) to your fish configuration directory preserving the directory structure.
+```console
+nvm install latest
+```
 
-```fish
-set -q XDG_CONFIG_HOME; or set XDG_CONFIG_HOME ~/.config
+Install the latest [LTS](https://github.com/nodejs/Release) (long-term support) Node release.
 
-for i in conf.d functions completions
-  curl https://git.io/$i.nvm.fish --create-dirs -sLo $XDG_CONFIG_HOME/fish/$i/nvm.fish
-end
+```console
+nvm install lts
 ```
 
-To uninstall nvm, just run:
+Install an older LTS release by codename.
 
-```
-rm -f $XDG_CONFIG_HOME/fish/{conf.d,functions,completions}/nvm.fish && emit nvm_uninstall
+```console
+nvm install carbon
 ```
 
-</details>
+> Installs `8.16.2`, the latest release of the Carbon LTS line.
 
-## Quickstart
-
-Download and switch to the latest Node.js release.
+Or install a specific version of Node.
 
 ```console
-nvm use latest
+nvm install v12.9.1
 ```
 
-> **Note:** This downloads the latest Node.js release tarball from the [official mirror](https://nodejs.org/dist), extracts it to <code>[\$XDG_CONFIG_HOME](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html#variables)/nvm</code> and modifies your `$PATH` so it can be used immediately. Learn more about the Node.js release schedule [here](https://github.com/nodejs/Release).
+> Supports full or partial version numbers, starting with an optional "v".
 
-Download and switch to the latest LTS (long-term support) Node.js release.
+The `nvm install` command activates the specified Node version only in the current environment. If you want to set the default version for new shells use:
 
-```console
-nvm use lts
+```fish
+set -U nvm_default_version v12.9.1
```

Comment:
Would a flag for `nvm use` and `nvm install` such as `--universal` be a bad or good idea?

Reply:
To set `nvm_default_version` for you? It would be an idea. But is it so hard to set `nvm_default_version` yourself?

We can definitely come back to this after 2.0 lands. 👍

Reply:
Yes, to set `nvm_default_version` for me. I think my main concern is discoverability. I.e. pressing `nvm <tab>` would show the option. Where only having a universal var that could be set is harder for users to find.

Reply:
The nice thing about using a uvar is that Fish users already knows how to set, query, and erase them—they're common—not to mention they're fishy (idiomatic Fish) too. If we introduce a flag like `use --set` or `use --persist` people will ask for an `--unset` or `--erase` flag just like in the original nvm there's an `nvm unalias default VERSION`.

> Where only having a universal var that could be set is harder for users to find.

Maybe we can supply a man page or simply add info about variables in `nvm --help`.

Reply:
Agree, adding it to `nvm --help` is a great compromise and solves the problem.